annotator-core/src/main/java/edu/ucr/cs/riple/core/adapters/NullAwayV0Adapter.java

```diff
@@ -79,18 +78,19 @@ public Fix deserializeFix(Location location, String[] values) {
   }
 
   @Override
-  public Error deserializeError(String[] values) {
+  public Error deserializeError(String[] values, FieldDeclarationStore store) {
     Preconditions.checkArgument(
         values.length == 10,
         "Expected 10 values to create Error instance in NullAway serialization version 0 but found: "
             + values.length);
     String encMember = !Region.getType(values[3]).equals(Region.Type.METHOD) ? "null" : values[3];
-    return new Error(
-        values[0],
-        values[1],
-        new Region(values[2], encMember),
-        offset++,
-        Location.createLocationFromArrayInfo(Arrays.copyOfRange(values, 4, 10)));
+    Location nonnullTarget =
+        Location.createLocationFromArrayInfo(Arrays.copyOfRange(values, 4, 10));
+    String errorType = values[0];
+    String errorMessage = values[1];
+    Region region = new Region(values[2], encMember);
+    // since we have no information of offset, we set all to zero.
```

Comment:
I don't see anything being set to 0 here?

Reply:
Sorry that comment was outdated, fixed it eb40bf33e22ffbaa689c585eabcfc02760c446ef